.github/codeql/codeql-config.yaml

```diff
@@ -0,0 +1,5 @@
+queries:
+  - uses: security-extended
+  - uses: security-and-quality
```

Comment:
not sure what queries LGTM was using, but these built-in but non-default ones seemed useful